Misc/NEWS.d/next/Library/2020-01-11-22-09-29.bpo-31485.DSvjsY.rst

```diff
@@ -0,0 +1 @@
+On the tkinter library, the widget.unbind(<Sequence>, funcid) call unbinds all bindings for <Sequence>, while documentation states that when funcid is given all other bindings remains untuched. Apparently this behaviour is not offered by tkinter itself and, in case of non-empty funcid argument value be given, should be obtained by, at first, unbinding all bindings and then bind again all the ones not having the given funcid. This is basically what the proposed patch does. It takes account of the modifications done into the unbind method docstring, as described by other PSF members on the issue tracker.
```

Comment:
This is one line of about 500 chars.  I think we usually comment with line breaks, but I don't know if it makes much difference.  In any case, this seems like more explanation than needed by users, who are the audience for news items.

The technical details can be appropriate for the commit message.  You can edit the initial message on the PR to include a suggested commit message (about the bpo link).